hack/ci-e2e-kind-migration.sh

```diff
@@ -39,3 +39,6 @@ make gardenlet-kind2-up
 
 # run test
 make test-e2e-local-migration
+
+make gardener-down
+make gardenlet-kind2-down
```

Comment:
how about stopping the kind clusters in the end?
```
make kind2-down
make kind-down
```
That's at least something kubernetes is doing too: [ref](https://github.com/kubernetes-sigs/kind/blob/e3a4cd8884aa6ffde81e39888db9fec7aa1c24e9/hack/ci/e2e-k8s.sh#L38)

Reply:
Ah right, thanks for reminding me. Was planning to do it as well initially :D

Reply:
I would have to add these to the `trap` call so that they get executed after exporting logs. Btw something that came to my mind is that exporting logs at the end of the script execution when everything succeeds will not include logs from gardener since the releases will be deleted. But I think that would be ok, right?

Reply:
Yes, we only care about logs/events when something went wrong :)